docs/_config.yml

```diff
@@ -23,6 +23,6 @@ redcarpet:
   extensions: ['with_toc_data', 'smart']
 exclude: [_docs]
 baseurl: http://samza.apache.org
-version: latest
+version: 0.13.1
```

Comment:
need '' marks.

Reply:
Note that we don't need change in the doc version since we never track the minor version docs in the release. We simply overwrite the previous 0.13.0 doc version w/ the new 0.13.1 under the same 0.13 version.